src/routes/__tests__/browse.spec.ts

```diff
@@ -123,3 +126,51 @@ describe("generateCollectionResponse", () => {
     expect(collectionsResponse.description).to.equal("All collections provided by PROV1");
   });
 });
+
+describe("addQueryParametersToItemLink", () => {
```

Comment:
One thing that can improve readability is adding whitespace (blank lines) between logical sections of the test:
- Group all const/let declarations together at the top.
- Add a blank line before calling the function you're testing.
- Add another blank line before your expect assertion.

```
it("will append collection search query parameters", async () => {
  const mockRequest = { ... } as Request;
  let stacCollection = generateSTACCollections(1)[0];
  stacCollection.links.push({ ... });

  addQueryParametersToItemLink(stacCollection, mockRequest);

  expect(stacCollection).to.have.deep.property("links", [ ... ]);
});
```